python/delta/tables.py

```diff
@@ -594,6 +594,27 @@ def addFeatureSupport(self, featureName: str) -> None:
         DeltaTable._verify_type_str(featureName, "featureName")
         self._jdt.addFeatureSupport(featureName)
 
+    @since(4.0)  # type: ignore[arg-type]
+    def dropFeatureSupport(self, featureName: str) -> None:
+        """
+        Modify the protocol to drop a supported feature. The operation always normalizes the
```

Comment:
```suggestion
        Modify the protocol to drop an existing supported feature. The operation always normalizes the
```

Nit: Similar to the `alterDeltaTableCommand`'s description.